packages/core-components/src/lib/molecules/toast/Toast.svelte

```diff
@@ -1,17 +1,27 @@
 <script>
+	import { createEventDispatcher } from 'svelte';
 	import { scale, fly } from 'svelte/transition';
 
+	export let id;
 	/** @type {import("@evidence-dev/component-utilities/stores).ToastStatus } */
 	export let status = 'info';
 	export let title;
 	export let message;
+	export let dismissable = true;
+	const dispatch = createEventDispatcher();
+
+	const dismiss = () => {
+		if (dismissable) dispatch('dismiss', { id });
+	};
 </script>
 
 <div
 	class="print:hidden rounded py-1 px-3 my-4 mx-0 shadow-md text-xs font-mono flex justify-between transition-all duration-300 border {status ??
 		''}"
 	in:scale
 	out:fly|local={{ x: 1000, duration: 1000, delay: 0, opacity: 0.8 }}
+	on:click={dismiss}
+	on:keypress={dismiss}
```

Comment:
Should this be on any keypress? Probably doesn't matter too much but could be somewhat unexpected

Reply:
I don't know how you'd focus a toast honestly; this is just to satisfy aria